apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -716,11 +716,11 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
         voteId = votesLength++;
 
         Vote storage vote_ = votes[voteId];
-        vote_.status = VoteStatus.Normal;
-        vote_.startDate = getTimestamp64();
-        vote_.snapshotBlock = snapshotBlock;
         vote_.totalPower = totalPower;
         vote_.settingId = _getCurrentSettingId();
+        vote_.startDate = getTimestamp64();
+        vote_.snapshotBlock = snapshotBlock;
+        vote_.status = VoteStatus.Normal;
```

Comment:
Note that this change to move `status` to be after `snapshot` block (to align with the struct), saves us 1.5k gas on new votes :)